etc/kayobe/ansible/requirements.yml

```diff
@@ -29,3 +30,4 @@ roles:
     version: 1.18.5
   - src: https://github.com/stackhpc/ansible-role-docker.git
     name: geerlingguy.docker
+    version: 0.0.1
```

Comment:
The only strange thing about this version is that has no bearing to the original versions on the role prefork: https://galaxy.ansible.com/ui/standalone/roles/geerlingguy/docker/versions/

Reply:
I'm wondering if we should follow  our stackhpc/<stackhpc major.><stackhpc minor>.<stackhpc patch>.<# stackhpc commits ontop> namescheme e.g https://github.com/stackhpc/kolla-ansible/tags, but agree that pinning is definitely a good idea

Reply:
Good idea, I've made a new tag and pinned to that

Reply:
Thanks for highlighting our use of a fork of geerlingguy.docker, I asked @jackhodgkiss if he could update his pull request: https://github.com/geerlingguy/ansible-role-docker/pull/423